docs/source/app-dev/command-deduplication.rst

```diff
@@ -229,7 +231,16 @@ Fields in the error metadata are written as ``field`` in lowercase letters.
        - Set the deduplication offset to ``earliest_offset`` or the deduplication duration to ``longest_duration`` and retry from :ref:`Step 2 <dedup-bounded-step-offset>`,  obtaining the completion offset ``OFF1``.
 	 This may lead to accepting the change twice within the originally intended deduplication period.
 
-	 
+
```

Comment:
```suggestion
```

Reply:
Not sure I follow, the standard here is to have 2 empty lines between elements.